frontend/src/features/admin-form/settings/SettingsPage.tsx

```diff
@@ -38,47 +36,31 @@ export const SettingsPage = (): JSX.Element => {
       navigate(`${ADMINFORM_ROUTE}/${formId}/${ADMINFORM_RESULTS_SUBROUTE}`)
   }, [formId, hasEditAccess, isCollabLoading, navigate])
 
-  const tabOrientation: UseTabsProps['orientation'] = useBreakpointValue({
-    base: 'horizontal',
-    xs: 'horizontal',
-    md: 'vertical',
-  })
-
   const { ref, onMouseDown } = useDraggable<HTMLDivElement>()
 
   return (
-    <Box
-      overflow="auto"
-      flex={1}
-      // Buffer for bottom navbar in mobile breakpoints.
-      mb={{ base: '4rem', md: 'initial' }}
-    >
+    <Box overflow="auto" flex={1}>
       <Tabs
         isLazy
         isManual
-        orientation={tabOrientation}
+        orientation="vertical"
         variant="line"
-        py={{ base: '2.5rem', md: '3.5rem', lg: '4rem' }}
-        px={{ base: '1.5rem', md: '1.75rem', lg: '2rem' }}
+        py={{ base: '3.5rem', lg: '4rem' }}
+        px={{ base: '1.75rem', lg: '2rem' }}
```

Comment:
may still need to update the padding here, since mobile padding is slightly less looking at the figma

Reply:
which part of the figma? :o

Reply:
ok, updated in [f084a00](https://github.com/opengovsg/FormSG/pull/4815/commits/f084a0043f9f3871c94b345adbcdfe7b4254bf9b). Recomputed the x and y paddings so it should look correct now